### ISSUE TITLE
Update README to include QueryableExtensions

### ISSUE THREAD

**Issue by jzabroski** [VariableNotFound/ef6-tagwith#7]
I think this is much nicer than the raw format you provide, and was the original intention for how to use EFCore TagWith.

```c#
public static class IQueryableExtensions
{
    public static IQueryable<T> TagWithSource<T>(this IQueryable<T> queryable,
        string tag = "",
        [CallerMemberName] string methodName = "",
        [CallerFilePath] string sourceFilePath = "",
        [CallerLineNumber] int line = 0)
    {
        return queryable.TagWith(string.IsNullOrEmpty(tag) 
            ? $"{methodName} - {sourceFilePath}:{line}" 
            : $"{tag}{Environment.NewLine}{methodName}  - {sourceFilePath}:{line}");
    }
}
```

**Comment by jmaguilar** [comment]
It looks great. I'll include something like that in the next update :)

Thanks for your suggestion!

**Comment by jmaguilar** [comment]
Included in version 1.2.0. Thanks!
